src/services/array-functions.js

```diff
@@ -21,7 +21,17 @@ export function map(theArray, fnc){
 //fnc will return true or false, if true add the item to the new array else do not
 //return the new array
 export function filter(theArray, fnc){
-
+    var shortNamesArray = [];
+    for(var i = 0; i < theArray.length; i ++){
+        var currentItem = theArray[i];
+        fnc(currentItem);
+        var returnedItem = fnc(currentItem);
+        if (returnedItem = true) {
```

Comment:
double check you code here.
how do you compare things to true or false in an if statement